### ISSUE TITLE
Allow some history instead of pure exclude

### ISSUE THREAD

**Issue by woulf** [the-black-eagle/script.database.cleaner#20]
Hello there,
bit rusty with github so that's no really an issue but a feature request.
Would it be possible instead of totaly excluding a path to keep the last N entries.
for exemple keeping the last 10 videos seen on youtube 
thanks

**Comment by the-black-eagle** [comment]
Yeah, this is a good idea and is something I intend to look at in the new year.  Code will need a major re-write though so I don't expect it'll be implemented in the first week or so, but it is something I would like to include.  https://forum.kodi.tv/showthread.php?tid=272112&pid=2802103#pid2802103
